zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -106,14 +109,29 @@ pub fn init_wallet_db<P>(wdb: &WalletDb<P>) -> Result<(), rusqlite::Error> {
         )",
         NO_PARAMS,
     )?;
+    wdb.conn.execute(
+        "CREATE TABLE IF NOT EXISTS utxos (
+            id_utxo INTEGER PRIMARY KEY,
+            address TEXT NOT NULL, 
+            prevout_txid BLOB NOT NULL, 
+            prevout_idx INTEGER NOT NULL, 
+            script BLOB NOT NULL, 
+            value_zat INTEGER NOT NULL, 
+            height INTEGER NOT NULL,
+            spent_in_tx INTEGER,
+            FOREIGN KEY (spent_in_tx) REFERENCES transactions(id_tx),
+            CONSTRAINT tx_outpoint UNIQUE (prevout_txid, prevout_idx)
+        )",
+        NO_PARAMS,
+    )?;
```

Comment:
Is it okay to run a client with the "transparent-inputs" feature enabled on some runs and disabled on others? Document the answer.

Reply:
fixed in https://github.com/zcash/librustzcash/pull/341/commits/f75ffb0eaf3cbb37a6b06cb8b0865c550159f9e8